### ISSUE TITLE
'/*.fc6-1' and '*_C.txt'

### ISSUE THREAD

**Issue by simon5u** [WaqasSultani/AnomalyDetectionCVPR2018#8]
Hi, could you please provide the examples of *.fc6-1 and *_C.txt? For your information, we are reproducing your experimental results, but we cannot find a way to use Save_C3DFeatures_32Segments.m without knowing the input. Thanks.

**Comment by simon5u** [comment]
I found an example in c3d, c3d_sport1m_feature_extraction_video.sh, problem solved!

**Comment by dolongbien** [comment]
> Hi!Could you tell me how would you solve the problem? Was files generated using Caffe?

Yes they are. After running command **c3d_sport1m_feature_extraction_video.sh**, check output folder specified in that bash **.sh** file. All *.fc6-1 (or fc7 based on your command) files will be in that folder. You can use that output folder as the input for your Matlab script.
